### DISCUSSION TITLE
ReFi Research (Regenerative Finance)

### DISCUSSION THREAD

**Discussion by rufuspollock** [rufuspollock/awesome-crypto-critique#184] [General]
ReFi seems to be on the most "progressive" areas in crypto space. Would be good to get some background and research on this.

* [ ] What is it?
* [ ] Who's involved in it?
* [ ] What are the themes, aspirations and claims?

Maybe start with stuff in Zotero? I started a folder here:

https://www.zotero.org/groups/4600269/web3/collections/6484MLF5

And started a first note on one of Jeff Emmet's pieces here https://web3.lifeitself.us/notes/emmett-2018-rewriting-story-human

**Comment by AndreasS2501** [comment]
Yes very much so! HI Rufus I put quickly some notes down below from my zk that I feel might be related. Feel free to ask questions about context or details.

Here is a interview between (kind of) ETH maximalist and somone quite open from the cosmos ecosystem - I like how he from the start kind of jumped to the opposite of their claims/value propositions:
# Bankless Episode 116 - Why Bankless Is Wrong | Ethan Buchman - Cosmos

- #tags: #blockchain #web3 #ethan_buchman #podcast #history #cross_reference #models #narrative #money #circular_economy #cycles #society
- #cross_reference: #jordan_hall #gavin_wood
- via https://twitter.com/koeppelmann/status/1521438789457129473 martin koeppelman : "Highly recommended!
Fully agree with @buchmanster  vision but we are trying to build towards it within the Ethereum ecosystem. 
 @CirclesUBI, @MEVprotection  and @gnosischain  together can create Ethans version of money and circular clearing."
- https://twitter.com/TrustlessState/status/1521208006838345730 quoted tweet
- https://www.youtube.com/watch?v=Vbomgw-PGRE
- Podcast Hosts: Ryan Sean Adams, David Hoffman
- Guest: Ethan Buchman Co- CEO Cosmos
- Some rough topic themes:
- 1. Multi-Chain Future - Will there be many chains or a few power-law winners?
- 2. Does Money Win? Does a chain need to be money to win?
- 3. Security - Ethereum Shared security or an App Chain Model?
- 4. Avoiding Dystopia - How can we prevent both Anarchy and Empire Dystopias?
- Ethan: there are many things that are not good about the current bank system
- but the point is about emphasis(basically ethan politely disputes most of the hosts views on money/crypto, which are quite superfical)
- **so rather than bankless - I(Ethan) would say CoOp Full**
- **rather then monetary premium - monetary clearing**
- **rather then empires - city states**
- **rather then DeFi decentralized finance - regenerative and collaborative finance**
- Ethan: perspective on history and medieval history
- Lex mercatoria Phenomena, I used in my blockchain 2018 talk
- Commons origin in th UK -> complexity 
- capitalism and italian city states
- Ethan: IBC standard cosmos Gavins XCM is it: https://medium.com/polkadot-network/xcm-part-ii-versioning-and-compatibility-b313fc257b83 
- Ethan - units of account in context of - the role of bitcoin
- Systems and cycles: roman to american empire .. these things progress <== under which definition of progress? Sand talk
- Q from 1:12:05: for ethan how do you "compete" vs the network effects of money : do not forget money is an abstraction
 jordan hall : https://twitter.com/jgreenhall/status/1495014877261406221
"The only true store of value is a thriving, evolvable  community in healthy relationship with thriving land that can maintain itself in relationship with all other human activities."
- My answer: you outcompete by removing abstraction by making the - local - reality impact more valuable than some abstract form of numbers value on a computer, a - m a d e - relaization of value, connection doesn't need abstraction (in the hyper sense)
- 1:15:00 Mutual exclusivity: (beard) the citxy states are part of the empire .... I think he meant: city states and empire are IN RELATIONSHIP
- Ethan: Layer 2 vs cosmos security: security as an obligation - reminds me on tysons perspective on it, its about soverinty and agency
- Ethan: there is more then the crypto economic secutiry on the line: YES its about context, and community
- (beard) gestalt psychology: sum is greater then it parts...
- topic cooperative democracies: flipping the economic models: proof of work, proof of stake ,proof of bandwisth, to holy grale: proof of care ,  proof of plant:
 **WE SHOULD BE ISSUING MONEY TO PEOPLE THAT KEEP SOCIETY ALIVE**:  the whole economic system needs to be flipped
 - proof of care : proof of social impact?
 - proof of engagement
 - chaos and order as potential elemts of dystopia
 - cosmos: try to surface power, how to surface power : nice blen into the existentialal crypto: transparency vs privacy
 - crypto banks lido
 - **cosmos** : limit on number of validators, I felt the whole conversation ethan had the better , slightly more wholesome perspectives but then ryan asked him about how come that cosmos validators are so fewe and how most-all of them are inside buddies !
 - Ethan MMT post for context: https://ebuchman.github.io/posts/mmt/

**Reply by rufuspollock** [reply]
@AndreasS2501 thanks very much for sharing. Do you have any analysis of how these claims actually make sense. Maybe we could build a claims tree so we can make sense of the claims and evaluate them.

**Reply by AndreasS2501** [reply]
I think its very difficult to verify these claims because you would need some kind of measurable unit. There are cases where money was actually used to create positive impact in the real world:
https://twitter.com/LuukDAO/status/1473787322508484627

I don't know how many other results like this exist. As far as I know the ImpactDAOs initiative from Kevin O.  is the best effort from inside the web3 space to create some kind of measurable impact metric.

**Reply by rufuspollock** [reply]
@AndreasS2501 not sure you need a measurable unit. Judges render judgments in court cases all the time based on reasoned evaluation of the evidence before them. As you know, i'm personally pretty sceptical based on all my research and evaluation so far.

Take the tweet you shared: what does that actually prove about ReFi? What is the claim sequence that links that to substantive potential impact for ReFi? Working through the chain or web of subclaims of an overall claim e.g.

- https://twitter.com/LuukDAO/status/1473787322508484627 #ReFi is not just a meme, it is actually our best chance at #regenerating earth and creating prosperity. 7 months ago this was a dry, empty plot of land. Now we have a 1000 sq m of syntropic agroforests. 
  - The factual claim is true: 7m ago this was dry empty land, now it is syntropic rainforest
  - This 1000m2 (10x100m) cost X to do (e.g. 10k euros or 10 euros / m2)
  - The scale of regeneration needed is 1000 million m2 (1000 km2 or 10 x 100 km)
  - Therefore at cost of 10k euros / 1000 m2 => 10 billion to do
  - ReFi so far has raised Z amount for actual investment into regeneration 
  - So ReFi would need to scale by A amount
  - How would ReFi scale?
     - Why are people investing in ReFi? X,Y,Z reason
     - Those investment methods are scalable e.g. they yield a competitive return OR they are donation like and there is evidence that people will donate at level XX

Does this give a sense of what i'm talking about?

For me basic sensemaking involves a rigorous, transparent way to evaluate claims, especially the fairly "big" claims being made about crypto and ReFi etc.

**Reply by AndreasS2501** [reply]
Hi @rufuspollock 

Yes I think one could do the calculation to arrive at some number/cost. But I think its not as as simple as putting a number on all the land that needs to be "saved" or owned.

Have you had yet the chance to look at the jaya klara brekke keynote? She ( and of course elsewhere) explicitly warns against the expansive nature of markets, the tokenization/financializaiton of everything. 

So this is one of the bigger problems that I see kind of unresolved or haven't really found any simple solutions so far. In capitalism there only exist things that are related to capital or measurement for that matter. Everything else we call "externalities" And if something we do creates a negative effect there  how do we relate to it?

In that sense I do not see web3 as a way to "simply" put a number on everything in the world such that the economy can finally consume/involve it all. 
The most hopeful route I have found so far is to create a new way to introduce important contexts.  I hope to explain now what I mean by this. 

Noam Chomsky here talks about the Magna Carta (Charter) but also about the Charter of the Forest. 
Where as the Magna Carta talks about about rights and stuff related to the individual, the Charter of the Forest talked about the preservation of the commons ( just a few sentences from his for commons context).

https://youtu.be/e_EgdShO1K8?t=2328

Let me introduce a further reference to the commons and then tie it back together to web3:
Ivan Illich states here: https://www.davidtinapple.com/illich/1983_silence_commons.html
""Commons" is an Old English word. According to my Japanese friends, it is quite close to the meaning that iriai still has in Japanese "Commons," like iriai, is a word which, in preindustrial times, was used to designate certain aspects of the environment. People called commons those parts of the environment for which customary law exacted specific forms of community respect. People called commons that part of the environment which lay beyond their own thresholds and outside of their own possessions, to which, however, they had recognized claims of usage, not to produce commodities but to provide for the subsistence of their households. The customary law which humanized the environment by establishing the commons was usually **unwritten**. It was unwritten law not only because people did not care to write it down, but **because what it protected was a reality much too complex to fit into paragraphs. The law of the commons regulates the right of way, the right to fish and to hunt, to graze, and to collect wood or medicinal plants in the forest.**"

Very short summary:
So we have these severe problems with the commons ( see tragedy of the commons , Ostrom etc..)  and we have this web3 thing claiming it can do something to mitigate for these problems.  But as we saw we want to avoid the "simple" approach of putting a number on everything ( tkenization/financialization). So can web3 still provide value? 

I do think it can because web3 is just a technology we can still choose to organize in a beneficial way to solve the problem at hand (this is what I refer to "context" above).  In my view this context has to be naturally very narrow, it has to tied to a certain piece of land and some people attending to the land or having a "real" relationship with the land.

So there are certain things people need to do to attend to the land, lets assume they are not self sufficient yet. Now were I see a web3/tech opportunity is that if the custodial people of the land ( which should eventually in some form or one other be every human on the planet) need money to do something for/in relationship with the land, we can ask what _kind_ of money would they accept to do so. 

Do they take money from other cooperatives? Do they use help from other people from other communities which is represented through tokens?
This is the spot where real experiments and sustainability happens.  I hope I was not too vague or long winded. Its difficult to talk about these things since we are not talking about something which is "normal" or common sense to every one.

Kevin O (see inquery from below) asked on twitter ( I think ) just about this question, 

Someone replied with this infographic: 
https://twitter.com/antonio_paglino/status/1538962946759479301
"natural capital-backed experiential regenerative yield farming" - IBC regional blockchain

I guess we have to get as close as possible to the actual land, how it changed and what was the role/relationship of these IBC regional blockchain in Changing/maintaining the land. 

I just remembered something, I think is important. All this stuff I wrote about the commons, which is this kind of hard to grasp by numbers thing. So the vehicle of crypto/web3 should help us to realize where _it doesn't make sense_ to try to tokenise/measre. It should help to come into a different relationship with the commons: to care about the commons without being able to measure or quantify everything in it.  

I hope this make sense! If I contradict myself or there is something unclear, please share!

**Comment by AndreasS2501** [comment]
Other interesting Refi Efforts: 
Green file Coin:
# Reimagining Public Goods: Building a Solarpunk Future - Scott Moore 

- #tags: #youtube #talk #narrative #solar_punk #web3 #elinor_ostrom #cooperatives #gitcoin #public_goods
- https://youtube.com/watch?v=HJ3CqbzO6Jk&feature=share
- Funding the commons online conference
- Scott Moore and Kevin Ock founded gitcoin
- narratives
- from cyberpunk to cypherpunk to solarpunk
- Ostrom Commons Management
- rochdale cooperativism
- escaping the tragedy of the Commons
- Slide: Solarpukn is our alternative to doomerism
- "We can take agency and ownership back over our institutions"
- "we can use new global coordination tools to fight for causes we believe in"
- "we can mint our own cooperative currencies to found our efforts"
- "we can design novel incentive mechanisms to be regen not degen"
- solar punk  redefines public goods
- public goods funding matrix : Kernel
- "**_web3 is solarpunk_**" :
- "web3 is not just speculative casinos.
Many of us are building new models to fundpublic goods; to adress long standing - inequalits

Regenerative economics allows us to create new cooperative systems that create positive externalities 

And to regen == solar punk"
- tools for progress:  digial signatures (ENS), decentralized storage IPFS, retro active public goods funding gitcoin, optimism , modular governance tooling - zodiac
- sign in with ethereum?
- we can choose to build a more solarpunk future

# Regen CryptoEconomics for Good - Juan Benet

- #tags: #web3 #crypto #industry #change #coordination # conference #talk #juan_benet #gitcoin
- https://www.youtube.com/watch?v=hRe-wS5uuvY&t=1052s
- Green pill book by Keven O from gitcoin: https://store.gitcoin.co/products/green-pill-book-digital-edition
- Paretotopian goal alignment
- from: Green Planet -> Green Industry -> Green Crypto -> Green filecoin -> Green Storage Providers
- Some interesting thoughts on public goods funding ( see green pill) and science funding BUT way to optimistic about mechanism design

**Comment by AndreasS2501** [comment]
Kevin Owocki  from gitcoin on impact DAOS -  the next goal for gitcoin it seems

# ImpactDAOs- The Building Block of Regen web3 - Kevin Owocki & Ale Borda 

- #tags: #web3 #daos #funding #experiments #values #change #economy #gitcoin
- gitcoin 
- https://www.youtube.com/watch?v=8rWX1PArkYs
- "we want to create a world where regen web3 augments every income"
- definition impact DAO: "any web3 project that creates positive externalities for the world"
- gregory landua founder of regen network8 8 kinds of capital:
- https://thegoodliferevival.com/blog/eight-forms-of-capital#
- http://www.appleseedpermaculture.com/8-forms-of-capital/
- impact daos, IDs from here on
- IDs examples: 
- celo   proof of stake mobile blockchain
- proof of humanity (see decracy earth santi siri) - UBI project 
- toucan - carbon credits project
- gitcoin - QF digital public goods (ecosystem software) ==> it might be interesting to see how much money from the current QF round goes into actualy real world regenerative efforts ( buying land to reforest  or similar)
- CoordinApe ultra tokeneconomics? trying to reward "unmeasurable" contributions?
- IDs scale by compounding(stack)
- eco system mapping above and side view at 11:11
- IDs:  proof of positive externalities, highest Leverage point, ecosystem ImpactDAOs
- the internet of values: we can embed our values into our economic system
- "we have an open canvas we can design our external reality" well.. there is a lot to ponder here its not that simple, think vlads change, but I think they simply want to emphasize on our ability to change something about the current sate/system
- IDs next steps: social - telegram conference, build a social movement
- proposed extension to gitcoin grants : grants 2.0 - ecosystem platform - proof of impact
- proof of impact: signatures , boosting signal of gitcoin grants?

I think in some sense Kevin is genuinely trying:
https://twitter.com/owocki/status/1538931164823506944
"how might the web3 community engineer a capital rotation away from projects with the best ponzinomics + towards the projects with the best real-world impact?"

For this comes nicely back to Ethans point about the importance of circulating money, as opposed to the hording - HODL types of money proposed more from the bitcoin space.  


This proposal in particular looks interesting: https://gitcoin.co/grants/4923/earthist-decentralize-hemp-seeds
